cmd/snap-preseed/main.go

```diff
@@ -0,0 +1,89 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/jessevdk/go-flags"
+)
+
+const (
+	shortHelp = "Prerun the first boot seeding of snaps in a chroot with a snapd seed."
+	longHelp  = `
+	Prerun the first boot seeding of snaps in an Ubuntu-based chroot directory
```

Comment:
I think this needs a rewrite. It reads as a techno-jargon that is hard to follow.
```
The prerun command improves performance of first boot seeding process by
pre-computing and caching the result of certain operations.
```

Reply:
The audience of this tool is not end users.

Reply:
We can try to make the sentences less clunky but no point hiding details.

Reply:
Maybe @degville can help a bit.

Reply:
Maybe 

```
The snap-preseed command takes a directory with an image filesytem including seed snaps (at /var/lib/snapd/seed) and pre-runs as much as possible of first boot snap installation to optimize it.
```

Reply:
@pedronis this is very nice, let's use it.

Reply:
I'm made a slight tweak suggestion to this below.